src/Explorer/Controls/Settings/SettingsSubComponents/ContainerPolicyComponent.tsx

```diff
@@ -102,11 +102,57 @@ export const ContainerPolicyComponent: React.FC<ContainerPolicyComponentProps> =
 
   return (
     <div>
-      <Pivot onLinkClick={onPivotChange} selectedKey={ContainerPolicyTabTypes[selectedTab]}>
+      <Pivot
```

Comment:
Is it similar to SettingsComponent? If so, it would be nice to avoid the duplication.